IOS/SwiftSeedProject/SwiftSeedProject/Infrastructure/Network/Extensions/DataRequest+Extensions.swift

```diff
@@ -30,13 +30,12 @@ extension DataRequest: ApiRequestProtocol {
     }
     
     static func sanitizeError(json: JSON) -> Result<JSON> {
-        if !json["error"].exists() {
+        if json["status"].stringValue == "ok" {
```

Comment:
The closure that calls this function has a reference to the response object, which has a statusCode property. On this method we just parse the error if exists